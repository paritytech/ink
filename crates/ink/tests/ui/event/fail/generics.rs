#[derive(ink::Event)]
pub struct Event<T> {
    #[ink(topic)]
    pub topic: T,
}

fn main() {}