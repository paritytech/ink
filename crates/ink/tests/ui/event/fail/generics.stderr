error: can only derive `Event` for structs without generics
 --> tests/ui/event/fail/generics.rs:2:18
  |
2 | pub struct Event<T> {
  |                  ^
